### PR TITLE
Specification adjustments

### PR DESCRIPTION
**Pull request by debskipawel**

### CODE REVIEW

File: BikeRentalApi/serializers/stationSerializer.py [hunk 1]
```diff
@@ -9,10 +9,11 @@ class StationSerializer(serializers.ModelSerializer):
...
     activeBikesCount = SerializerMethodField()
+    bikesLimit = IntegerField(source = 'bikes_limit')
 
     class Meta:
         model = BikeStation
-        exclude = ['state']
+        exclude = ['state', 'bikes_limit']
 
     def get_activeBikesCount(self, obj: BikeStation):
         return obj.bike_set.filter(bike_state = BikeState.Working).count()
```
Comment (chlebnymaciej) [inline BikeRentalApi/serializers/stationSerializer.py:19]: Missing bike_limit

**Review by chlebnymaciej — CHANGES_REQUESTED** [review]
Please make this change

File: BikeRentalApi/serializers/stationSerializer.py (hunk 1 above)
Comment (debskipawel) [reply BikeRentalApi/serializers/stationSerializer.py:19]: Sure thing, doing it right now!

**Review by chlebnymaciej — APPROVED** [review]
Looks good now

**Comment by debskipawel** [comment]
> Looks good now

**Pull request merged**